src/TTImp/TTImp.idr

```diff
@@ -790,6 +790,7 @@ definedInBlock ns decls =
 
     defName : Namespace -> ImpDecl -> List Name
```

Comment:
Should this be a `Set` to avoid too many duplicates?

Reply:
Yeah, it should.  I got here via `Local.idr`, which calls `nub` on the result, but I see it is called from two other locations that do not do that.  I don't see `nubOrd` in the library, is `(SortedSet.toList . SortedSet.fromList) $` ok as a makeshift `nubOrd` or should I thread a `SortedSet` through the function?  (I could also call `nub`, but it's O(n^2).)

Reply:
My instinct would be to thread a `Set` and then keep `definedInBlock`'s type unchanged so
that the call sites do not need to be modified (or only modified to remove calls to `nub`).